accounts/src/inmemory_account_store.rs

```diff
@@ -74,12 +78,14 @@ impl AccountDataByCommitment {
             .map(|x| x.updated_slot < data.updated_slot)
```

Comment:
the naming `updated_slot` is strange. what does it mean in context of the struct?
should only be called `slot`

Reply:
When was the account last updated.